### DISCUSSION TITLE
Does Umami work without a domain (only the server IP)?

### DISCUSSION THREAD

**Discussion by giuliomagnifico** [umami-software/umami#1358] [Q&A]
Hi,
I just installed Umami on DigitalOcean via docker. But it’s running in a droplet without a domain name or https and it doesn’t collect any stats. 

I’ve put two Umami tracker codes in my website, one on the instance on DigitalOcean and one on Heroku with the domain. 

The one on Heroku works, the other not. 

So I just wonder if it’s my configuration issue or Umami tracker code doesn’t work when inside there’s an IP address instead of the domain name. 

Thanks

**Comment by owmsoton** [comment]
Hi,

i'm using umami with IP and over HTTP and it works fine, I'm on version V1.33.0

**Comment by giuliomagnifico** [comment]
Okay thanks. Then it’s my fault, I’ll try to reinstall it.
